ibmcloud/terraform/cos/main.tf

```diff
@@ -0,0 +1,29 @@
+data "ibm_resource_group" "group" {
+  name = "default"
+}
+
+########## Create a COS instance
+resource "ibm_resource_instance" "cos_instance" {
+  name              = "cos-image-instance"
```

Comment:
This is a good default name.  I think it is convenient if the instance name is configurable. We can define a variable for this name in variables.tf, and a user can customize it in terraform.tfvars.

Reply:
The user can now enter a COS instance name into a `terraform.tfvars` file and Terraform will use it